src/hubAuth.ts

```diff
@@ -50,7 +82,7 @@ export const testkitHubAuth = (homeDir: string): void => {
     );
     return;
   }
-  if (getAuthStrategy() === AuthStrategy.AUTH_URL && process.env.TESTKIT_AUTH_URL) {
+  if (getAuthStrategy() === AuthStrategy.AUTH_URL) {
```

Comment:
`getAuthStrategy` uses the env var to determine result. No need to include the env var again here.

Reply:
Replaced `process.env.` with `env.getString()` throughout.